roles/cis/tasks/3/3.1.1.yml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Does this call sysctl command add the entry to sysctl.conf file if value is not the expected value and do a flush? Please refer this https://gitlab.eng.vmware.com/core-build/photon-hardening/commit/cdb23e5ab12bee7fba5213894ba53583400073d0 incase you have not seen the previous script implementaion on how we handle all sysctl based configurations and we generally write the photon specific sysctl configs to /etc/sysctl.d/50-security-hardening.conf not to /etc/sysctl.conf just to differentiate what is been set by photon hardening.

Reply:
Thanks for the review.
unfortunately in chroot, sysctl commands are throwing errors:
```shell
    if [ "$1" == "both" ]; then
       sysctl -w $IPROUTEFLUSH > /dev/null
       sysctl -w $IPV6ROUTEFLUSH > /dev/null
    elif [ "$1" == "ipv4" ]; then
       sysctl -w $IPROUTEFLUSH > /dev/null
    elif [ "$1" == "ipv6" ]; then
       sysctl -w $IPV6ROUTEFLUSH > /dev/null
    fi
```
I will add another block for executing shell commands in localhost.
In localhost the sysctl shell commands are working.

recommendation was to use sysctl module which by default is writing to /etc/sysctl.conf:
https://docs.ansible.com/ansible/latest/modules/sysctl_module.html
I will investigate to write to /etc/sysctl.d/50-security-hardening.conf

Reply:
you can change the file destination of sysctl module. 
as @keerthanakalyan points out, the verification of this rule will only verify file contents if done in chroot. this is okay. please add comment in verification stating the same.
verification for production should be done via inspec or similar.

Reply:
Done, the changes are made to apply the configuration to /etc/sysctl.d/50-security-hardening.conf and verify the same